test/suite_implementation.js

```diff
@@ -30,13 +31,15 @@ module.exports = function(style, options, _callback) {
 
     const map = new Map({
         container: container,
-        style: style,
         classes: options.classes,
         interactive: false,
         attributionControl: false,
         preserveDrawingBuffer: true
     });
 
+    const _style = new Style(style, map, { validate: !options.ignored });
+    map.setStyle(_style);
+
```

Comment:
We can use `skipped` flags to get around validating these tests without adding new code